### ISSUE TITLE
What are the expected performance gains when using model quantization? \ Is there support for keras models?

### ISSUE THREAD

**Issue by ophiry** [tensorflow/tensorflow#18026]
[this page](https://www.tensorflow.org/performance/quantization) describes tensorflow model quantization 

* Is there a similar tool for keras models?
* are there some examples of the performance gain I can expected? ()it's probably model dependent, but some examples would be usefull

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jart** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
